pyphare/pyphare/pharein/__init__.py

```diff
@@ -283,8 +283,11 @@ def as_paths(rb):
 
         add_string(restarts_path + "filePath", restart_file_path)
 
+        if "elapsed_timestamps" in restart_options:
+            pp.add_array_as_vector(restarts_path + "elapsed_timestamps", restart_options["elapsed_timestamps"])
 
-        pp.add_array_as_vector(restarts_path + "write_timestamps", restart_options["timestamps"])
+        if "timestamps" in restart_options:
+            pp.add_array_as_vector(restarts_path + "write_timestamps", restart_options["timestamps"])
```

Comment:
`"timestamps"` should maybe be renamed

Reply:
@nicolasaunai thoughts on updating the original (non-elapsed time based) variable from `timestamps` to something clearer?

Reply:
oh you answered already below 

> I think indeed the term "timestamps" should be reserved for the "simulation times" and here maybe clock_timestamps (or other?) should be used for being clear we're talking about true wall clock time.